xbmc/android/activity/XBMCApp.cpp

```diff
@@ -299,6 +301,38 @@ bool CXBMCApp::EnableWakeLock(bool on)
   return true;
 }
 
+bool CXBMCApp::EnableAudioFocus(bool on)
```

Comment:
It's a valid point, but we already have EnableWakeLock(), so I much prefer consistency in this instance